Gems/PhysX/Code/Source/PhysXCharacters/API/CharacterUtils.cpp

```diff
@@ -95,6 +95,7 @@ namespace PhysX::Utils::Characters
 
             controllerDesc.scaleCoeff = extendedConfig.m_scaleCoefficient;
             controllerDesc.contactOffset = extendedConfig.m_contactOffset;
+            controllerDesc.position = PxMathConvertExtended(extendedConfig.m_position);
```

Comment:
Could you move this line to the `AppendShapeIndependentProperties` function above since position isn't PhysX specific?